src/plugins/intel_npu/src/plugin/npuw/util.hpp

```diff
@@ -99,6 +99,15 @@ struct Impl {
         return iter->second;
     }
 
+    template <typename K>
+    V& at_or_at_or_at(const K& k1, const K& k2, const K& k3) {
```

Comment:
This may continue forever, seems we need to introduce some monadic wrapper for `at`. :)

Reply:
OR a variadic template could save the day here. BTW.